ocs_sample_library_preview/BaseClient.py

```diff
@@ -97,32 +97,22 @@ def sdsHeaders(self) -> dict[str, str]:
         if (self.__auth_object is not None):
             headers['Authorization'] = 'Bearer %s' % self._getToken()
         if (self.__accept_verbosity):
-            headers['Accept-Verbosity'] = 'verbose'
+            # All possible routes should call the same verbosity header function to ensure case sensitivity
+            # accept-verbosity and Accept-Verbosity would not overwrite each other, leading to unpredicable response from OCS
+            headers.update(BaseClient.getVerbosityHeader(True))
         if self.__request_timeout is not None:
             headers['Request-Timeout'] = str(self.__request_timeout)
 
         return headers
 
-    def communityHeaders(self, community_id: str):
```

Comment:
removing public functions off of BaseClient would be moving this to .6 (even if we expect people not to be using them directly).   I might suggest adding the new methods and maybe keeping these but marking them as obsolete and suggesting the right function or something